packages/liquidator/test/Liquidator.js

```diff
@@ -679,9 +769,15 @@ contract("Liquidator.js", function(accounts) {
 
       await liquidator.update();
       await liquidator.liquidatePositions(amountToLiquidate);
-      assert.equal(spy.callCount, 3); // 1 info + 2 error level events should be sent at the conclusion of the 1 successful (incl. 1 partial) and 1 failed liquidations.
-      assert.equal(spyLogLevel(spy, 2), "error");
-      assert.isTrue(spyLogIncludes(spy, 2, "Failed to liquidate position"));
+
+      // 5 error + 2 info
+      assert.equal(spy.callCount, 5);
+      assert.equal(spyLogLevel(spy, 4), "error");
+      assert.isTrue(spyLogIncludes(spy, 4, "Failed to liquidate position"));
+      assert.equal(spyLogLevel(spy, 3), "warn");
+      assert.isTrue(spyLogIncludes(spy, 3, "exchange return amount too low"));
```

Comment:
So why does this fail, is it because the bot doesn't have enough reserve currency to convert into synthetic?

Reply:
It'll fail because there is no volume on the exchanges to facillitate the trade